pages/_app.js

```diff
@@ -21,6 +21,7 @@ export default function MyApp({ Component, pageProps }) {
       bannerContent={pageProps.bannerContent}
       popupContent={pageProps.popupContent}
       display={display}
+      lang={pageProps.locale}
```

Comment:
Can this be removed? Its the same as the line above. `locale={pageProps.locale}`